### ISSUE TITLE
Switch to PHPunit legacy version until PHP version in Virtualbox is updated

### ISSUE THREAD

**Issue by benlk** [INN/deploy-tools#45]
https://phar.phpunit.de/phpunit-old.phar retains support for PHP 5.3 that was dropped in https://phar.phpunit.de/phpunit.phar. The virtualbox bootstrap scripts currently install the new version, which requires PHP 5.6 or greater: https://github.com/INN/deploy-tools/blob/15b722a4a95f1ecda7b0d2beeed2bd086787d1ff/vagrant/boot-script.sh#L119-L122

```sh
# Install PHPUnit for wp unit testing.
wget https://phar.phpunit.de/phpunit.phar
chmod +x phpunit.phar
mv phpunit.phar /usr/local/bin/phpunit
```

- [ ] change url
- [ ] update in umbrella repos

**Comment by benlk** [comment]
To downgrade a newly-built box: 

```sh
wget https://phar.phpunit.de/phpunit-old.phar
chmod +x phpunit-old.phar 
mv phpunit-old.phar  phpunit
which phpunit
```
/usr/local/bin/phpunit
```
sudo mv phpunit /usr/local/bin/phpunit
```

**Comment by benlk** [comment]
Marking this closed and wontfix because of the plan in #48.
